dashboard/package.json

```diff
@@ -150,7 +150,10 @@
     "tslint": "^5.9.1",
     "tslint-config-prettier": "^1.18.0",
     "tslint-react": "^4.1.0",
-    "typescript": "^4.1.0"
+    "typescript": "^4.2.3"
+  },
+  "resolutions": {
+    "@typescript-eslint/typescript-estree": "^4.17.0"
```

Comment:
Since tslint has been deprecated , the ts version was not being properly caught. So I forced the resolution to the latest version with support for 4.2.3

See https://github.com/palantir/tslint/issues/4534